.github/workflows/ci.yaml

```diff
@@ -45,6 +45,76 @@ jobs:
           override: true
 
       - uses: actions-rs/cargo@v1
+
+      - uses: actions-rs/install@v0.1
+        with:
+          crate: wasm-snip
+          version: latest
+  
+      - name: Installing binaryen
+        run: sudo apt-get install binaryen
+        
+      - name: Setting up golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.0'
+
+      - name: Installing golang dependencies
+        run: |
+          go version
+          go get -u golang.org/x/lint/golint
+          go get github.com/stretchr/testify/assert
+
+      - name: Building application
+        run: make build
+
+  # dependent job added to run after build if it succeeds      
+  integration:
+    needs: build
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust:
+          - stable
+        experimental: [false]
+        cargo_flags: [--all-features]
+    steps:
+      - uses: actions/checkout@v2
         with:
-          command: build
-          args: --target=wasm32-unknown-unknown
+          submodules: recursive
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          components: rustfmt
+          target: wasm32-unknown-unknown
+          override: true
+
+      - uses: actions-rs/cargo@v1
+      - uses: actions-rs/install@v0.1
+        with:
+          crate: wasm-snip
+          version: latest
+      - name: Installing binaryen
+        run: sudo apt-get install binaryen
+        
+      - name: Setting up golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.15.0'
```

Comment:
Probably not very important, but note you are using 1.16 elsewhere in the go module configuration.

Reply:
Sorry I have several golang versions installed on my machine and was working with 1.16. Will change the CI to 1.16.